content/python/concepts/sets/terms/update/update.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.update()'
+Description: 'Updates the current set, by adding items from another iterable(set, list, tuple or dictionary), removing any duplicates.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+The `.update()` method updates the current [set](https://www.codecademy.com/resources/docs/python/sets), by adding items from another iterable(set, list, tuple or dictionary), removing any duplicates.
```

Comment:
```suggestion
The **`.update()`** [method](https://www.codecademy.com/resources/docs/java/methods) updates the current [set](https://www.codecademy.com/resources/docs/python/sets), by adding items from another iterable(set, list, tuple or dictionary), removing any duplicates.
```

Reply:
Modifications:

Left a line before the sentence.
The first instance of the term should be wrapped in bold+backticks: **`.update()`**.